refnx/reflect/_mdsimulation.py

```diff
@@ -0,0 +1,307 @@
+        if self.verbose:
+            print('PDB file read.')
+        self.av_layers = np.zeros((int(np.floor(self.dimensions[2] /
+                                                layer_thickness)) + 1, 5))
+        self.av_layers[:, 0] = layer_thickness
+        self.av_layers[:, 3] = layer_thickness * roughness
+        self.layers = np.array([self.av_layers, ] * len(self.structure))
+        self.flip = flip
+        self.cut_off = cut_off
+
+    def assign_scattering_lengths(self, lgtfile=None, radiation='neutron',
+                                  xray_energy=None, atom_types=None,
+                                  scattering_lengths=None):
+        """
+        Assign the scattering lengths either from a .lgt file, a user defined
+        array or allow it to be determined from the elements listed in the
+        .pdb file.
+        Parameters
+        ----------
+        radiation: str, optional
+            Either 'neutron' or 'xray'. This is the type of radiation that was
+            used in the experimental reflectometry measurements.
+        xray_energy: float, optional
+            Only required if the radiation is 'xray'. This is the energy of the
+            probing X-ray radiation, in units of kilo electron Volts
+        lgtfile: str, optional
+            The path and name of the .lgt file (if present), which contains the
+            scattering lengths of each of the atom type in the pdbfile.
+            Currently the .lgt file style that is supported is a 3 column space
+            separated txt file where the columns are atom_type,
+            real_scattering_length, and imaginary_scattering_length
+            respectively. If a lgtfile is not used the scattering lengths for
+            the system will be determined based on the element type that is
+            defined in the final column of the pdb file.
+        atom_types: str, array_like
+            An array of length N, where N is the number of atom types in the
+            simulation trajectory, which defines the atom types to assign
+            scattering lengths.
+        scattering_lengths: float, array_like
+            An array of shape [N, 2], where N is the number of atom types in
+            the simulation trajectory, which defines the scattering lengths for
+            the given atom types.
+        """
+        self.scatlens = {}
+        if atom_types and scattering_lengths:
+            if len(atom_types) == len(scattering_lengths):
+                for i in range(0, len(atom_types)):
+                    self.scatlens[atom_types[i]] = scattering_lengths[i]
+            else:
+                raise ValueError('The lengths of the atom types must be the '
+                                 'same as the lengths of the pairs of '
+                                 'scattering lengths.')
+        elif lgtfile:
+            self.lgtfile = lgtfile
+            self.read_lgt()
+        else:
+            if radiation == 'neutron':
```

Comment:
I think this should work.
```
self.neutron = radiation == 'neutron'
```

Reply:
how would this would with the else statement below (I'm not familiar with that syntax)